packages/explorer/src/app/(explorer)/queries/useDeployedTablesQuery.ts

```diff
@@ -0,0 +1,57 @@
+import { useParams } from "next/navigation";
+import { Hex } from "viem";
+import { anvil } from "viem/chains";
+import mudConfig from "@latticexyz/store/mud.config";
+import { useQuery } from "@tanstack/react-query";
+import { internalNamespaces } from "../../../common";
+import { DeployedTable, decodeTable } from "../api/utils/decodeTable";
+import { useChain } from "../hooks/useChain";
+import { useIndexerApiUrl } from "../hooks/useIndexerApiUrl";
+import { DozerResponse } from "../types";
+
+export function useDeployedTablesQuery() {
+  const { worldAddress, chainName } = useParams();
+  const { id: chainId } = useChain();
+  const indexerApiUrl = useIndexerApiUrl();
+
+  return useQuery<DozerResponse, Error, DeployedTable[]>({
+    queryKey: ["deployedTables", worldAddress, chainName],
+    queryFn: async () => {
+      const storeTablesKey = "store__Tables";
+      const tableName = chainId === anvil.id ? `${worldAddress}__${storeTablesKey}` : storeTablesKey;
+      const query =
+        chainId === anvil.id
+          ? `SELECT * FROM "${tableName}"`
+          : `SELECT ${Object.keys(mudConfig.tables[storeTablesKey].schema).join(", ")} FROM ${tableName}`;
```

Comment:
this is one of those areas I am worried about the chain ID getting sprinkled around
```suggestion
      const tableName = "store__Tables";
      const query =
        indexer.type === 'sqlite'
          ? `SELECT * FROM "${worldAddress}__${tableName}"`
          : `SELECT ${Object.keys(mudConfig.tables[tableName].schema).join(", ")} FROM ${tableName}`;
```

Reply:
Yeah see your point, like this improvement, updated